File: MATEMÁTICA/1429/main.py
def decimal(acm):
    factorial = [1, 1, 2, 6, 24, 120]
    result = 0
    i = 1
    while acm > 0:
        digit = acm % 10
        result += digit * factorial[i]
        i += 1
        acm //= 10
    return result


while acm := int(input()):
    print(decimal(acm))

File: MATEMÁTICA/1429/main.js
function decimal(acm) {
    let factorial = [1, 1, 2, 6, 24, 120];
    let result = 0;
    for (let i = 1; acm > 0; i++) {
        let digit = acm % 10;
        result += digit * factorial[i];
        acm = parseInt(acm / 10);
    }
    return result;
}

let input = require('fs').readFileSync('/dev/stdin', 'utf-8');
let lines = input.split('\n');
let i = 0;
let acm;
while (acm = parseInt(lines[i++])) {
    console.log(decimal(acm));
}
